certbot/plugins/manual.py

```diff
@@ -42,7 +42,8 @@ class Authenticator(common.Plugin):
 
 {validation}
 
-Once this is deployed,"""
+Once this TXT record is deployed,
+Let's Encrypt will be able to check the validity of your domain."""
```

Comment:
nit: The protocol that Certbot and Let's Encrypt uses (called ACME) is an open standard and there are other organizations starting to adopt it. Because of this, Certbot can be used with more servers than Let's Encrypt so I wouldn't mention them explicitly here. You could change this to "the ACME server", "the ACME CA", "the certificate authority", etc.

Personally though, I have a slight preference for a different approach. In our instructions for the HTTP challenge, we say:
```
Make sure your web server displays the following content at <url> before continuing:
```
We could do something similar here, delete the "Once this is deployed" line and change the beginning of the prompt to say "Before continuing, please deploy...". This is an aesthetic thing though and if you like your approach better, we can go with it as long as you remove the string "Let's Encrypt".